src/AppBundle/Controller/SelfRegisterController.php

```diff
@@ -48,6 +49,42 @@ public function register(Request $request)
         return $user;
     }
 
+
+    /**
+     * @Route("/verifycodeputy")
+     * @Method({"POST"})
+     */
+    public function verifyCoDeputy(Request $request)
+    {
+        $coDeputyVerified = false;
+
+        if (!$this->getAuthService()->isSecretValid($request)) {
+            throw new \RuntimeException('client secret not accepted.', 403);
+        }
+
+        $selfRegisterData = new SelfRegisterData();
+        $this->populateSelfReg($selfRegisterData, $this->deserializeBodyContent($request));
+
+        $validator = $this->get('validator');
+        $errors = $validator->validate($selfRegisterData, ['verify_codeputy']);
+
+        if (count($errors) > 0) {
+            throw new \RuntimeException('Invalid registration data: ' . $errors);
+        }
+
+        try {
+            $coDeputyVerified = $this->container->get('user_registration_service')->validateCoDeputy($selfRegisterData);
+            $this->get('logger')->warning('CasRec codeputy validation success: ', ['extra' => ['page' => 'codep_validation', 'success' => true] + $selfRegisterData->toArray()]);
+        } catch (\Exception $e) {
+            $this->get('logger')->warning('CasRec codeputy validation failed:', ['extra' => ['page' => 'codep_validation', 'success' => false] + $selfRegisterData->toArray()]);
+throw $e;
+        }
+
+return print_r(doctrineDebug::export($coDeputyVerified, 3),true);
+
+        return ['verified' => $coDeputyVerified];
```

Comment:
above 6 lines need a reformat. remove last return statement

Reply:
that was debug! good spot